backend/templates/emails/reminder-email.html

```diff
@@ -0,0 +1,84 @@
+{% extends 'emails/base.html' %}
+
+{% block content %}
+<h2>Reminder to Renew <b>{{ name }}</b> on Penn Clubs & SAC Fair Signup by 8/24!</h2>
+<p style="font-size: 1.2em">
+    Dear <b>{{ name }}</b> leaders,
+</p>
+
+<p style="font-size: 1.2em">
+    You are receiving this notification because you were the contact for your organization during the 2019 SAC
```

Comment:
Since this email will only be sent to the club officers who have not yet renewed their club, we can add something like this to the top.

> You have not yet renewed your club for the {{ year }} school year. If you do not submit a renewal request by August 24th, your club may not appear on the Penn Clubs website in time for new student orientation. Use the green button below to renew your club. The process takes less than 5 minutes.

You can also make this specific text red to bring attention to it, like in [this email](https://pennclubs.com/api/emailpreview/?email=approval_status&approved=False).

This way, we can make it explicit as to what exactly they have to do if they don't want to read the whole email, since the email is quite long.